instantout/actions.go

```diff
@@ -165,11 +167,17 @@ func (f *FSM) InitInstantOutAction(eventCtx fsm.EventContext) fsm.EventType {
 	}
 
 	// Create the address that we'll send the funds to.
-	sweepAddress, err := f.cfg.Wallet.NextAddr(
-		f.ctx, "", walletrpc.AddressType_TAPROOT_PUBKEY, false,
-	)
-	if err != nil {
-		return f.HandleError(err)
+	var sweepAddress btcutil.Address
```

Comment:
Ultra-mirco nit, could we simplify to

```suggestion
        var sweepAddress = initCtx.sweepAddress
	if sweepAddress == nil {
		sweepAddress, err = f.cfg.Wallet.NextAddr(
			f.ctx, lnwallet.DefaultAccountName,
			walletrpc.AddressType_TAPROOT_PUBKEY, false,
		)
		if err != nil {
			return f.HandleError(err)
		}
	}
```